src/Resources/config/orm.php

```diff
@@ -42,7 +43,7 @@
             ->args([
                 param('sonata.page.page.class'),
                 service('doctrine'),
-                service('sonata.page.slugify.cocur'),
```

Comment:
let's also mark this service as deprecated?

Reply:
Deprecated!